### DISCUSSION TITLE
Inpaint brush not selecting anything

### DISCUSSION THREAD

**Discussion by phyfutima** [lllyasviel/Fooocus#2856] [Q&A] [answered]
Loving fooocus; the only problem I'm having right now is with inpainting, which is that I can't inpaint at all. When I paint with the inpaint brush, it doesn't actually select anything. It feels like a bug, but I think if it were a bug it would have been fixed by now because everyone would be reporting it! So it must be something I'm not doing right. Does this sound familiar to anyone -- when I paint with the inpaint brush, as soon as I let go of the mouse the "selection" that I made with the brush immediately disappears, and so nothing is actually selected and nothing can get inpainted? Any idea what I'm doing wrong?

**Comment by ILikeToasters** [comment]
What are you using Fooocus on? Which browser? Because I would start with that being the problem potentially. Or could be browser settings? Test in a different browser if you can to start with.

**Reply by phyfutima** [reply, marked as answer]
Thank you for the suggestion! It turns out my browser (Epic) wasn't the problem, but its encrypted proxy was on, and that was the problem -- I turned that off, and the inpainting started working. Thanks again for pointing me in the right direction!
